otx/algorithms/action/configs/classification/x3d/template.yaml

```diff
@@ -0,0 +1,64 @@
+# Description.
+model_template_id: Custom_Action_Classificaiton_X3D
+name: X3D
+task_type: ACTION_CLASSIFICATION
+task_family: VISION
+instantiation: "CLASS"
+summary: Basic transfer learning template for X3D
+application: ~
+
+# Algo backend.
+framework: OTEDetection v2.9.1
+
+# Task implementations.
+entrypoints:
+  base: otx.algorithms.action.tasks.ActionClsTrainTask
+  openvino: otx.algorithms.action.tasks.OpenVINODetectionTask
```

Comment:
This is a Classification template, does it use the Openvino Detection task?

Reply:
Currently there is no nncf and openvino tasks for action classification. It will be added after this PR merged
At that PR this will be changed

Reply:
If it is something that cannot be used in this PR, it would be good to delete it first.